prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics-release.yaml

```diff
@@ -16,7 +16,7 @@ prow_ignored:
     serviceAccountName: e2e-test-runner
     containers:
     - &config-sync-ci-container
-      image: us-docker.pkg.dev/kpt-config-sync-ci-artifacts/test-infra/gke-e2e:v1.0.0-590be013d
+      image: us-docker.pkg.dev/kpt-config-sync-ci-artifacts/test-infra/gke-e2e:v1.0.0-57875fa5
```

Comment:
What commit SHA is this built from? Ideally this tag should be built from a SHA on `main`

Reply:
Built from https://github.com/GoogleContainerTools/kpt-config-sync/pull/875. 
It's tricky because if we merge that PR first without updating the gke-e2e image, the periodic jobs will fail for that PR.

Reply:
Or we can split https://github.com/GoogleContainerTools/kpt-config-sync/pull/875 by updating gke-e2e to go 1.20 first, and then updating the remaining, and split this PR by updating kubekins-e2e and gke-e2e separately.

That requires PRs to be submitted in the following order:
1. PR1: update oss-test-infra with a new kubekins-e2e image, which is compatible with the new go version (to make presubmit check pass).
2. PR2: update the golang version in the Dockerfile of gke-e2e first, and merge the change to main (to get the change on the main branch first, so we can use the commit SHA to build gke-e2e image).
3. build and push the gke-e2e image from the commit on main.
4. PR3: update oss-test-infra with the new gke-e2e image  (to make sure periodic jobs won't break for step 5).
5. PR4: update kpt-config-sync go version
6.

Reply:
There shouldn't be any issues with periodic jobs failing, we just need to make the infra available before upgrading. I would suggest this order:
1. Submit change to kpt-config-sync to bump the Go version used by the test-infra images
2. Publish new test-infra images
3. Bump images used by the presubmit/periodic prowjobs in oss-test-infra
4. Proceed with upgrading the Go version used by Config Sync

Reply:
That merged PR1 and PR3 together. That sounds good.

Reply:
Updated to only update kubekins-e2e image

Reply:
Done. The gke-e2e image was built based on the commit on the main branch of kpt-config-sync:
https://github.com/GoogleContainerTools/kpt-config-sync/commit/f11bb971f413a6c9ee0e11378568b841c717b23e.